WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -212,7 +213,22 @@ public void onPageSelected(int position) {
 
             @Override
             public void onPageScrollStateChanged(int state) {
-                // noop
+                if (!mWasSwiped && state == ViewPager.SCROLL_STATE_DRAGGING) {
+                    mWasSwiped = true;
+                }
+
+                if (mWasSwiped && state == ViewPager.SCROLL_STATE_IDLE) {
+                    mWasSwiped = false;
+
+                    switch (AppPrefs.getMainTabIndex()) {
+                        case WPMainTabAdapter.TAB_MY_SITE:
+                        case WPMainTabAdapter.TAB_READER:
+                        case WPMainTabAdapter.TAB_ME:
+                        case WPMainTabAdapter.TAB_NOTIFS:
+                        default:
+                            AnalyticsTracker.track(AnalyticsTracker.Stat.MAIN_TABS_SWIPED);
```

Comment:
Can we simply drop this `switch` block, since no matter what we're tracking the same thing?

Reply:
Also, I noticed that we track even if the user swiped and let go of the same page (so the page doesn't actually change). Do we want to track that?

Reply:
> Can we simply drop this `switch` block, since no matter what we're tracking the same thing?

Yes, we can, but I intentionally used it to easily differentiate between each tab as I mentioned in the pull request description.

> The `switch` statement was included in order to easily add analytics for each tab specifically if needed in the future.

This is only meant to be a short-term change, but it may be kept a little longer if we want to get some analytics on each tab.

> Also, I noticed that we track even if the user swiped and let go of the same page (so the page doesn't actually change). Do we want to track that?

Yes, this is meant to just track swiping in general.

Reply:
> Yes, we can, but I intentionally used it to easily differentiate between each tab as I mentioned in the pull request description.

Sorry, I really need to stop speed-reading PR descriptions.